src/dbms/handler.hpp

```diff
@@ -120,6 +154,7 @@ class Handler {
  private:
   std::unordered_map<std::string, utils::Gatekeeper<T>, string_hash, std::equal_to<>>
       items_;  //!< map to all active items
```

Comment:
What do these items_ do, is there better naming for it?

Reply:
This is a generic handler that stores anything. 
Can't think of a better name. `objects` seems as vague as `items` :shrug: